models/public/densenet-169-tf/densenet-169-tf.md

```diff
@@ -32,6 +32,7 @@ group of models designed to perform image classification. The weights were conve
     - C - number of channels
 
    Expected color order - BGR.
+   Mean values - [123.68, 116.78, 103.94], scale factor for each channel - 58.8235294
```

Comment:
@eizamaliev ,  should we align scale factor format? 58.8235294 or 1/58.8235294?

Reply:
Model Convertor uses `--scale_values` flag, which matches 58.8235294 format.

Reply:
Agree. In description section second option preferable, for consistency with descriptiption from [here](https://software.intel.com/en-us/articles/model-downloader-essentials).
But this section have more common with MO parameters.
Best option is move mean and scale to descrtiption section.

Reply:
Can I resolve this conversation?

Reply:
Yes